src/zenml/client.py

```diff
@@ -5516,20 +5515,27 @@ def get_model(
 
     def list_models(
```

Comment:
Still think this method should be kept consistent with other `client.list_X` methods by
- accepting pagination and filtering params
- return `Page` instead of `List` again

Reply:
Addressed